docs/tutorials/creating-a-runtime-module.md

```diff
@@ -193,4 +193,25 @@ And run your node with
 Finally, start the [Apps UI](https://polkadot.js.org/apps/#/explorer) to confirm that the module is working as expected. If you aren't familiar with the Apps UI, you will need to navigate to the settings tab, and select 'Local Node'.
 
 ## Customize your Module
-In this tutorial you've learned to create a runtime module as a Rust crate. You're now ready to replace the template module logic with your own logic. If you're not sure what to code, check out one of our runtime modules in [tutorials](https://substrate.dev/en/tutorials).
+So far in this tutorial you've learned to create a runtime module as a Rust crate. You're now ready to replace the template logic with your own. If you're not sure what to code, check out one of our runtime modules in [tutorials](https://substrate.dev/en/tutorials).
+
+## Publish your Module
+Once your module is no longer just a template, you should consider publishing it. We'll cover publishing your module to github here, but crates.io is another good option. Go ahead and [create a github repository](https://help.github.com/en/articles/create-a-repo) and [push your module's code](https://help.github.com/en/articles/pushing-to-a-remote) to it.
+
+With the code on github, your module is properly published. Congratulations! We now need to update your node to use the code that is on github instead of a hard-coded file system path.
+
+The final edit to your runtime's `Cargo.toml` file will update its dependency on your module. The new code is
+```toml
+[dependencies.test_module]
+default_features = false
+git = "https://github.com/<you>/<your-module>"
```

Comment:
```suggestion
git = "https://github.com/<username>/<repository>"
```

Reply:
You are missing specifying the `package`. If it happens that your module is named `test_module`, then this will work as is, but in general, probably better to be explicit about package name here.